include/boost/beast/websocket/impl/read.hpp

```diff
@@ -227,8 +227,6 @@ class stream<NextLayer, deflateSupported>::read_some_op
                     // Handle ping frame
                     if(impl.rd_fh.op == detail::opcode::ping)
                     {
-                        impl.update_timer(this->get_executor());
```

Comment:
Updating the timer is necessary; otherwise, it will attempt to ping the other side even after receiving a ping.